### PR TITLE
Reconfigure package with zopefoundation/meta/config

### PR DESCRIPTION
**Pull request by dataflake**

### CODE REVIEW

**Review by jugmac00 — APPROVED** [review]
LGTM

File: .meta.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+with-pypy = false
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[tox]
+use-flake8 = true
+
+[coverage]
+fail-under = 65
```
Comment (icemac) [inline .meta.toml:18]: Previously we had 75 % coverage (see setup.cfg) – was this simply wrong or did the coverage actually drop?
GHA counts 66 % (see https://github.com/zopefoundation/Products.ZSQLMethods/runs/2107041611?check_suite_focus=true)

**Review by icemac — APPROVED** [review]
LGTM but I have a single question.

File: .meta.toml (hunk 1 above)
Comment (dataflake) [reply .meta.toml:18]: The reason is simple: On `master` the test runner only visits `src/Products/ZSQLMethods` and doesn't visit the path `src/DC/Shared/ZRDB`. The old configuration was broken.

**Pull request merged**